app/controllers/campaigns_controller.rb

```diff
@@ -48,6 +48,7 @@ def edit
 
   def create
     @campaign = Campaign.new(campaign_params)
+    @campaign.country_codes = @campaign.country_codes.filter { |code| code.present? }
```

Comment:
This is great but consider `@campaign.country_codes.select(&:present?)` to be a bit more idiomatic.